integration/update_test.go

```diff
@@ -96,13 +96,13 @@ func TestMultiFlag(t *testing.T) {
 				update:         bson.D{{"$set", bson.D{{"v", int32(43)}}}},
 				multi:          false,
 				stat:           bson.D{{"n", int32(1)}, {"nModified", int32(1)}, {"ok", float64(1)}},
-				expectedToFind: 5,
+				expectedToFind: 6,
```

Comment:
Created and issue to move update tests to compat https://github.com/FerretDB/FerretDB/issues/1610